extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/security/PersistentLoginManager.java

```diff
@@ -100,6 +104,11 @@ public RestoreResult restore(RoutingContext context) {
     }
 
     public void save(SecurityIdentity identity, RoutingContext context, RestoreResult restoreResult, boolean secureCookie) {
+        save(identity.getPrincipal().getName(), context, cookieName, restoreResult, secureCookie);
```

Comment:
@stuartwdouglas Stuart, can you please review these `PersistenLoginManager` changes ?

Reply:
Not sure this is a big change, I'm just allowing this to be reused for other cookie names.

Reply:
It is fine, my concern is about a case where both WebAuthn and Form mechanisms can coexist; but lets address it before considering a status change from `preview`